src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add: (a, b) => {
+      return a + b;
+    },
+    subtract: (a, b) => {
+      return a - b;
+    },
+    multiply: (a, b) => {
+      return a * b;
+    },
+    divide: (a, b) => {
+      return a / b;
+    },
```

Comment:
```suggestion
    add(a, b) {
      return a + b;
    },
    subtract(a, b) {
      return a - b;
    },
    multiply(a, b) {
      return a * b;
    },
    divide(a, b) {
      return a / b;
    },
```

Reply:
Let's use one approach in one object (half of your methods are arrow functions, half are function declarations)